package.json

```diff
@@ -7,7 +7,8 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\"",
     "test": "npm run lint && npm run build",
     "build": "webpack -p",
-    "start": "webpack-dev-server -d --inline --hot"
+    "start": "webpack-dev-server -d --inline --hot",
+    "prepublish": "babel -d dist/ src/"
```

Comment:
An alternative would be to use CommonJS for now and stick with ES5: no need for that `prepublish` calling `babel` (and no need for the `babel-cli` dev dependency), just update the `main` to point to `src/Main.js` and add `dist/` to `.npmignore`.